src/linuxdeploy.cpp

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+#include <boost/filesystem/path.hpp>
+
+#include <linuxdeploy/core/appdir.h>
+#include <linuxdeploy/core/log.h>
+#include <linuxdeploy/util/util.h>
+
+#include "linuxdeploy.h"
+
+using namespace linuxdeploy::core;
+
+using namespace linuxdeploy::core::log;
+using namespace linuxdeploy::util;
+namespace bf = boost::filesystem;
+
+namespace linuxdeploy {
+    desktopfile::DesktopFile getMainDesktopFile(std::vector<std::string>& desktopFilePaths,
```

Comment:
Forward declaration is unnecessary -- just move the other function above. Forward declarations are the root of all evil.

Reply:
fiexed